extensions/amp-install-serviceworker/amp-install-serviceworker.md

```diff
@@ -73,6 +73,69 @@ URL of a HTML document that install a ServiceWorker.
 
 Must have the value `nodisplay`.
 
+### `data-no-service-worker-fallback-url-match`
+
+A regular expression that matches URLs to be rewritten to navigate via shell for no-service-worker fallback.
+See [Shell URL rewrite](#shell-url-rewrite) section for more details.
+
+The value must be a valid JavaScript RegExp string, for instance:
+ - `amp.html`
+ - `.*amp`
+ - `.*\.amp\.html`
+ - `.*\/amp$`
+
+### `data-no-service-worker-fallback-shell-url`
+
+The URL to the shell to use to rewrite URL navigations for no-service-worker fallback.
+See [Shell URL rewrite](#shell-url-rewrite) section for more details.
+
+The value must be an URL on the same origin as the AMP document itself.
+
+## Shell URL rewrite
+
+When service workers are not available or not yet active, it's possible to configure URL rewrite to direct
+navigations to the shell. This way, e.g. AMP Runtime can redirect navigation to the "shell" instead of
+a "leaf" AMP document.
+
+This fallback is only used when the document is opened on the source origin, and NOT on proxy origin.
+
+The URL rewrite is configured using `data-no-service-worker-fallback-url-match` and `data-no-service-worker-fallback-shell-url`
+attributes as following:
+```
+<amp-install-serviceworker layout="nodisplay"
+      src="https://www.your-domain.com/serviceworker.js"
+      data-no-service-worker-fallback-url-match=".*\.amp.html"
+      data-no-service-worker-fallback-shell-url="https://pub.com/shell">
+</amp-install-serviceworker>
+```
+
+Here:
+ - `data-no-service-worker-fallback-url-match` specifies the link for AMP+PWA shell. It's required to be on the source origin as the AMP document.
```

Comment:
Typo `data-no-service-worker-fallback-url-match` should be `data-no-service-worker-fallback-shell-url`